modules/client/src/test/java/org/apache/ignite/client/fakes/FakeTxManager.java

```diff
@@ -179,6 +181,12 @@ public CompletableFuture<Void> cleanup(ClusterNode recipientNode, List<IgniteBiT
         return null;
     }
 
+    @Override
+    public CompletableFuture<Void> cleanup(ClusterNode recipientNode, TablePartitionId tablePartitionId, Long term, UUID txId,
```

Comment:
Is it possible to use node consistent Id instead of cluster node as a recipient one?

Reply:
In that case I'd rather use it.